README.md

```diff
@@ -1,38 +1,41 @@
 # gvisor-tap-vsock 
 
-A replacement for [VPNKit](https://github.com/moby/vpnkit), written in pure Go.
-It is based on the network stack of [gVisor](https://github.com/google/gvisor/tree/master/pkg/tcpip).
-
-## How it works
 
-### Internet access
+A replacement for [libslirp](https://gitlab.com/qemu-project/libslirp) and [VPNKit](https://github.com/moby/vpnkit), written in pure Go.
+It is based on the network stack of [gVisor](https://github.com/google/gvisor/tree/master/pkg/tcpip).
 
-![schema](./doc/curl.png)
+Compared to libslirp, gvisor-tap-vsock brings:
+* a configurable DNS server,
```

Comment:
given there are only two things in the list, i would recommend the use of a sentence over a list.

Reply:
yes done :+1: